### ISSUE TITLE
一个view反复加载一张图，内存一直增长，没有释放

### ISSUE THREAD

**Issue by javavaan** [penfeizhou/APNG4Android#78]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | e.g.  1.0.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by penfeizhou** [comment]
请给出具体的测试场景及代码

**Comment by javavaan** [comment]
`private void loadApng(ImageView imageView, int res, final long startTime) {
        ResourceStreamLoader resourceStreamLoader = new ResourceStreamLoader(MainActivity.this, res);
        final APNGDrawable apngDrawable = new APNGDrawable(resourceStreamLoader);

        apngDrawable.setLoopLimit(1);
        imageView.setImageDrawable(null);
        imageView.setImageDrawable(apngDrawable);
        apngDrawable.registerAnimationCallback(new Animatable2Compat.AnimationCallback() {
            @Override
            public void onAnimationStart(Drawable drawable) {
                super.onAnimationStart(drawable);
                System.out.println("duration: " + (System.currentTimeMillis() - startTime));
            }

            @Override
            public void onAnimationEnd(Drawable drawable) {
                super.onAnimationEnd(drawable);
            }
        });
    }`

每隔1s执行一次
`loadApng(iv2, R.drawable.zanapng, 0);`

**Comment by penfeizhou** [comment]
内存能释放的前提是触发GC
你这段测试代码是在重复创建Drawable,没有复用图片,内存肯定是在增长的
请这段代码执行完以后手动GC后看内存有没有释放.

**Comment by penfeizhou** [comment]
实测不存在内存泄漏问题,issue关闭
